sdks/python/apache_beam/__init__.py

```diff
@@ -66,7 +66,7 @@
 import sys
 
 
-if sys.version_info.major != 2:
+if sys.version_info[0] != 2:
```

Comment:
Python SDK is not compatible with python < 2.7. It seems like this check was coincidentally ensuring that. I agree with your change, since we need to produce a better error message. Could you also check that `minor` version is at least 7.